package.json

```diff
@@ -68,7 +68,7 @@
     "node-sass": "3.4.2",
     "page": "1.6.4",
     "path-parser": "1.0.2",
-    "phone": "git+https://github.com/Automattic/node-phone.git#1.0.4-8",
+    "phone": "git+https://github.com/Automattic/node-phone.git#8f012b153e968038c2ce758b7f34e5b8d792eb20",
```

Comment:
This looks like a funny version to point at. The latest published version is 1.0.5. 
Calling:
```
npm install --save phone@1.0.5 
```
Should update package.json to
```
"phone": "1.0.5",
```